### PR TITLE
Update discord.py

### PR DESCRIPTION
**Pull request by extreme4all**

### CODE REVIEW

File: SQL.py [hunk 1]
```diff
@@ -479,3 +479,16 @@ def get_player_report_locations(players):
 
     data = execute_sql(sql, param=param, debug=True, has_return=True)
     return data
+    
+def get_region_search(regionName):
+
+    sql = "SELECT * FROM regionIDNames WHERE region_name LIKE :region"
+
+    regionName = "%" + regionName + "%"
```
Comment (extreme4all) [inline SQL.py:487]: this should be in the sql not in the variable, i think
`sql = "SELECT * FROM regionIDNames WHERE region_name LIKE '%:region%'"`

**Review by extreme4all — COMMENTED** [review]
possible improvement

File: discord/discord.py [hunk 2]
```diff
@@ -32,4 +33,28 @@ def get_locations(token):
 
     return jsonify(output)
 
+@discord.route('/discord/region/<token>', methods=['GET', 'POST'])
```
Comment (extreme4all) [inline discord/discord.py:36]: are you going to give multiple region's or just one?

File: SQL.py (hunk 1 above)
Comment (Ferrariic) [reply SQL.py:487]: Yes, I wasn't sure if that was possible with flask alch, good improvement.

File: discord/discord.py (hunk 2 above)
Comment (Ferrariic) [reply discord/discord.py:36]: Multiple at the moment !region acts like a search function, the most likely value is processed client side as the result, and is used in !map !heatmap

**Pull request closed without merging**